lib/diagnostic.rb

```diff
@@ -108,10 +108,17 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'Andy = Person.new(':favorite_number', ':first_name')'
+
+class Person
+  def initiatialize (num, name)
+    @favorite_number = num
+    @first_name = name
+  end
+end
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'Andy[:first_name] = 'Andy''
```

Comment:
Remember we need to use interpolation here.

`'person.default = "#{person[:first_name]} is great"'`